common/src/main/java/org/apache/uniffle/common/config/RssConf.java

```diff
@@ -532,9 +532,9 @@ public String getValue(ConfigOption<?> configOption) {
   // --------------------------------------------------------------------------------------------
 
   @Override
-  public RssConf clone() {
-    RssConf config = new RssConf();
-    config.addAll(this);
+  public RssConf clone() throws CloneNotSupportedException {
+    RssConf config = (RssConf) super.clone();
```

Comment:
Why do we need this change?

Reply:
https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#cn-clone-method-does-not-call-super-clone-cn-idiom-no-super-call

> This non-final class defines a clone() method that does not call super.clone(). If this class ("A") is extended by a subclass ("B"), and the subclass B calls super.clone(), then it is likely that B's clone() method will return an object of type A, which violates the standard contract for clone().
>
> If all clone() methods call super.clone(), then they are guaranteed to use Object.clone(), which always returns an object of the correct type.

Reply:
```
RssConf config = (RssConf) super.clone();
```
The code seems a little strange.

Reply:
In fact, this is the Cloneable idiom. If a subclass extends this class and doesn't override `clone()`, it will still be correct.

Reply:
OK.